### ISSUE TITLE
Use cosmwasm std mocking in tests

### ISSUE THREAD

**Issue by d3v3us** [ezekiiel/cw-ibc-example#1]
We could use mocking IBC functions below to  demonstrate IBC working example that comes with some cool example tests.
mock_ibc_channel, 
mock_ibc_packet_recv
mock_ibc_channel_connect_ack, 
mock_ibc_channel_connect_confirm etc.

**Comment by 0xekez** [comment]
Yes! Would love some help with this if you'd like to contribute. The cw-plus icw20 contract has some [nice examples](https://github.com/CosmWasm/cw-plus/blob/8fd29040ad9a38fb9bf43e5382e799b6d586fe6c/contracts/cw20-ics20/src/test_helpers.rs) of doing this.
